open-craft/opencraft
# BB-1876: Fix code quality issues

This PR fixes code quality issues and enables quality check on builds.

**Testing instructions:**
1. Review changes and make comments.
2. Check that all CI checks are green.
3. (Optional) Add random spaces/tabs/enters on a few ts and tsx files. Run `npm run lint` and check the errors.
4. (Optional) Run `npm run prettier` and check that the code was fixed, and no errors appear when running `npm run lint`. 

**Reviewer:**
